src/netex_idf/offers.rs

```diff
@@ -1089,6 +1091,7 @@ mod tests {
                 "Trip Short Name",
                 vehicle_journey.short_name.as_ref().unwrap()
             );
+            assert_eq!(Some("tp_id".to_string()), vehicle_journey.trip_property_id);
```

Comment:
```suggestion
            assert_eq!("tp_id", vehicle_journey.trip_property_id.as_ref().unwrap());
```

Reply:
why do you think it's better?

Reply:
It's easier to read the expected part which to me is the important part.

Reply:
I don't agree but as it's the same thing I will modify it